src/cpp/session/modules/SessionPythonEnvironments.cpp

```diff
@@ -52,12 +53,27 @@ void onPrefsChanged(const std::string& /* layerName */,
    }
 }
 
+void onDeferredInit(bool)
```

Comment:
Will this be in time to make sure it gets onto the `PATH` of terminals that are initialized at or near to startup?

Reply:
I _think_ it will be OK, previous terminals don't rehydrate until displayed. Definitely worth testing.